javageci-core/src/main/java/javax0/geci/accessor/Accessor.java

```diff
@@ -1,9 +1,11 @@
 package javax0.geci.accessor;
 
+import javax0.geci.annotations.Geci;
 import javax0.geci.api.Segment;
 
 import java.lang.reflect.Field;
 
+@Geci("annotationBuilder")
```

Comment:
It could but this should not depend on it, because generator specific annotations should be separate from the generators themselves and entirely optional.

Reply:
I removed the annotation because I'm unsure about the direction of the module. Will the dependency on javageci-annotation be phased out or...?

Reply:
There is no reason to phase it out. When a project uses Java::Geci there has to be a test scope dependency on Java::Geci engine, API and probably tools. The extra dependency on the annotation module with test scope does not harm.

Reply:
AnnotationBuilder now has and uses its own anonotation.